### PR TITLE
Nightwatch.js - add page objects

### PR DESCRIPTION
**Pull request by alecmerdler**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://nightwatchjs.org/guide#using-page-objects
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by Schlesiger** [comment]
Approved; but it looks like you need to resolve the merge conflict. The last PR added `export` to all the interfaces: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nightwatch/index.d.ts

File: types/nightwatch/index.d.ts [hunk 1]
```diff
@@ -4,8 +4,10 @@
 //                 Connor Schlesiger <https://github.com/schlesiger>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export interface NightwatchCustomPageObjects {
-    page: {};
+/* tslint:disable:max-line-length */
+
+export interface NightWatchCustomPageObjects {
+    page: {[name: string]: () => NightWatchBrowser};
```
Comment (alecmerdler) [inline types/nightwatch/index.d.ts]: Now that I've looked at the source code, I'm not sure this is the correct interface for Nightwatch Page Objects (https://github.com/nightwatchjs/nightwatch/wiki/Page-Object-API).

Comment (Schlesiger) [reply types/nightwatch/index.d.ts]: To be honest I am not all that familiar with Nightwatch. I can look at any additions to your PR and take your word for it. If you'd like to become a maintainer of this definition I will approve that as well.

Comment (alecmerdler) [reply types/nightwatch/index.d.ts]: I have no intention of becoming maintainer of this definition, but I will gladly revise this PR to more accurately reflect the actual JS interface.

**Pull request merged**